packages/ffe-datepicker-react/src/calendar/Calendar.js

```diff
@@ -163,16 +162,27 @@ export default class Calendar extends Component {
     }
 
     renderDate(date, index) {
-        if (date.isLead) {
-            return <LeadDate key={date.date} date={date} />;
+        const { calendar } = this.state;
+
+        if (date.isNonClickableDate) {
+            return <NonclickableDate key={date.date} date={date} />;
```

Comment:
Linje 167 har `NonClickable` (med stor C) og linje 168 har `Nonclickable` (med liten c). Usikker på hvilken som blir mest riktig (kanskje `Nonclickable`?), men det viktigste er at vi holder oss til én konvensjon :)

Reply:
Tror jeg går for NonClickabel siden det på engelsk enten er non clickabel eller non-clickable

Reply:
`NonClickable` isåfall :wink: